src/web/timeline.rkt

```diff
@@ -178,6 +179,28 @@
 (define (average . values)
   (/ (apply + values) (length values)))
 
+(define (render-phase-mixed-sampling mixsample)
+  `((dt "Precisions")
+    (dd (details
+         (summary "Click to see histograms")
+         ,@(for/list ([rec (in-list (group-by first mixsample))])
+             
+             (define n (random 100000))
+             (let ([op (car (car rec))]
+                   [precisions (map second rec)]
+                   [times (map third rec)])
+               
+               `(details
+                 (summary "Operation " ,(~a op) ", total time spent: " ,(~a (round (apply + times))) "ms")
```

Comment:
```suggestion
                 (summary "Operation " (code ,(~a op)) ", total time spent: " ,(~a (round (apply + times))) "ms")
```

Reply:
Also use `format-time` for the time amount, which will automatically choose the unit (ms, s, min) appropriately.

Reply:
It would also be very cool to put the % of total time across all records.